webapp/lib/controller.dart

```diff
@@ -293,6 +295,8 @@ void command(UIAction action, Data actionData) {
 }
 
 void updateNeedsReplyCharts(List<model.NeedsReplyData> selectedProjectNeedsReplyDataList) {
+  if (selectedProjectNeedsReplyDataList.isEmpty) return;
```

Comment:
What happens if we change the filter to a project that has no data (yet)? I think adding this line means that the charts won't be cleared up anymore, which I'm not sure this is what we want to happen?

Reply:
I've realized when one selects a project which is not a Needs Reply one, for example, it throws a bunch of exception because this method gets called even when you're selecting a System Events project

Reply:
ok, that makes sense. Can you please check though that this doesn't cause any problems when switching to a project that doesn't have any data for the most recent filtered period? E.g. set time filter to 8 days on the default project, and then switch to the COVID19-SOM project - does it show correctly the empty charts?

Reply:
I've seen the problem now. It's not being cleared when I change to the COVID19-SOM. I can remove the checks but do you have a suggestion of how to deal with the null exceptions being thrown.
```
dart_sdk.js:5172 Uncaught Error: Bad state: No element
    at Object.throw_ [as throw] (dart_sdk.js:3880)
    at _JSMapIterable.new.reduce (dart_sdk.js:14819)
    at Object.updateNeedsReplyCharts (controller.dart:322)
    at Object.command (controller.dart:290)
    at view.ChartFiltersView._internal.filterChartsByPeriod (view.dart:273)
    at view.dart:204
    at Object._checkAndCall (dart_sdk.js:4084)
    at Object.dcall (dart_sdk.js:4089)
    at HTMLSelectElement.<anonymous> (dart_sdk.js:97075)
```

Reply:
I think I have a fix in mind for preventing this. Lemme attempt it first